### PR TITLE
rasterize - fix timeout issue

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: 
[XSUP-42986](https://jira-dc.paloaltonetworks.com/browse/XSUP-42986)
[XSUP-42805](https://jira-dc.paloaltonetworks.com/browse/XSUP-42805)
[XSUP-43435](https://jira-dc.paloaltonetworks.com/browse/XSUP-43435)

## Description
- Added the **blocked_urls** parameter with default value `cloudflare.com` so the user can define which domains he want to block the requests
- Fixed an issue where the ***rasterize*** command failed to parse multiple URLs.
- Updated the Docker image to: *demisto/chromium:130.0.6723.115353*.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 1]
```diff
@@ -235,6 +235,14 @@ def network_request_will_be_sent(self, documentURL, **kwargs):
         '''Triggered when a request is sent by the browser, catches mailto URLs.'''
         demisto.debug(f'PychromeEventHandler.network_request_will_be_sent, {documentURL=}')
         self.is_mailto = documentURL.lower().startswith('mailto:')
+
+        request_url = kwargs.get('request', {}).get('url', '')
+        request_id = kwargs.get('requestId', '')
+        if "cloudflare" in request_url:
+            self.tab.Fetch.enable()
```
Comment (tkatzir) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:242]: Can't `enable` be called once, when starting the tab?

Need to `disable` afterwards?

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 2]
```diff
@@ -235,6 +236,26 @@ def network_request_will_be_sent(self, documentURL, **kwargs):
         '''Triggered when a request is sent by the browser, catches mailto URLs.'''
         demisto.debug(f'PychromeEventHandler.network_request_will_be_sent, {documentURL=}')
         self.is_mailto = documentURL.lower().startswith('mailto:')
+
+        request_url = kwargs.get('request', {}).get('url', '')
+
+        if any(value in request_url for value in BLOCKED_URLS):
+            self.tab.Fetch.enable()
+            demisto.info('Issuing of requestPaused events enabled.')
```
Comment (tkatzir) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:244]:
```suggestion
            demisto.debug('Fetch events enabled.')
```

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 3]
```diff
@@ -235,6 +236,26 @@ def network_request_will_be_sent(self, documentURL, **kwargs):
...
+
+
+
+    def handle_request_paused(self, **kwargs):
+        request_id = kwargs.get("requestId")
+        request_url = kwargs.get("request").get("url")
+
+        if any(value in request_url for value in BLOCKED_URLS):
```
Comment (tkatzir) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:252]: Need to check that it's a `redirect` request that you're blocking.
e.g. we don't want to block images.

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 4]
```diff
@@ -235,6 +236,26 @@ def network_request_will_be_sent(self, documentURL, **kwargs):
...
+            self.tab.Fetch.failRequest(requestId=request_id, errorReason="Aborted")
+            demisto.info(f"Request paused: {request_url=} , {request_id=}")
```
Comment (tkatzir) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:254]:
```suggestion
            demisto.debug(f"Request paused: {request_url=} , {request_id=}")
```

File: Packs/rasterize/Integrations/rasterize/rasterize.py [hunk 5]
```diff
@@ -235,6 +236,26 @@ def network_request_will_be_sent(self, documentURL, **kwargs):
...
+            self.tab.Fetch.disable()
+            demisto.info('Issuing of requestPaused events disabled.')
```
Comment (tkatzir) [inline Packs/rasterize/Integrations/rasterize/rasterize.py:256]:
```suggestion
            demisto.debug('Fetch events disabled.')
```

**Review by tkatzir — COMMENTED** [review]
Good progress.
Added comments.

File: Packs/rasterize/ReleaseNotes/2_0_26.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Rasterize
+
+- Added the parameter **List of domains to block** to allow an adjustment to block requests from specific domains.
+- Fixed an issue where ***rasterize*** command failed to parse multiple urls.
```
Comment (richardbluestone) [inline Packs/rasterize/ReleaseNotes/2_0_26.md:7]: Fixed an issue where the ***rasterize*** command failed to parse multiple URLs.

File: Packs/rasterize/ReleaseNotes/2_0_26.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Rasterize
+
+- Added the parameter **List of domains to block** to allow an adjustment to block requests from specific domains.
```
Comment (richardbluestone) [inline Packs/rasterize/ReleaseNotes/2_0_26.md:6]: Added the **List of domains to block** parameter, which allows an adjustment to block requests from specific domains.

**Review by richardbluestone — APPROVED** [review]

**Comment by adi88d** [comment]
Force merged due to unrelated build issues.

**Pull request merged**